platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/TextField.kt

```diff
@@ -126,8 +126,16 @@ public fun TextField(
                     )
                 }
             } else {
```

Comment:
Nit: now that the `if` has both branches, we can invert it so it reads `if (undecorated) { new stuff } else { old stuff }`